src/msg.jl

```diff
@@ -2,6 +2,18 @@ import Base.show
 
 export Msg, msg_pub, msg_reply, send_status, send_ipython
 
+# Julia commit 77631ce7f615059c7021756ef539d36964af13dc
+if VERSION >= v"0.5-dev+4194"
+    import Base.unsafe_string
+else
+    const unsafe_string = Base.bytestring
+end
+
+if length(methods(unsafe_string, (ZMQ.Message,))) == 0
+    unsafe_string(zmsg::ZMQ.Message) = unsafe_string(pointer(zmsg), length(zmsg))
+end
```

Comment:
This should be patched in ZMQ.jl, not here.  Until ZMQ is patched, we can live with the deprecation message.